projects/step-frontend/src/lib/modules/timeseries/modules/_common/types/time-series/time-series-context-params.ts

```diff
@@ -7,6 +7,7 @@ import { TsFilteringSettings } from '../filter/ts-filtering-settings';
 export interface TimeSeriesContextParams {
   id: string;
   timeRange: TimeRange;
+  attributes?: Record<string, MetricAttribute>; // attributes of all dashlets by their ids
```

Comment:
I'm not a fan of comments, probably this is selfexplanatory, else we should call it dashletAttributes